docusaurus/website/src/pages/styles.module.css

```diff
@@ -42,5 +42,8 @@
 
 .lightBackground {
   background-color: #f7f7f7;
-  color: rgb(31, 34, 39);
```

Comment:
Is the removal of `color` intentional or should it be keept for light mode?

Reply:
I believe its intentional so that the color will inherit from docusaurus own css. If this isnt defined, the color will always be this value in light/dark mode

Reply:
Yeah, I think it was only necessary because without it the font would be white-on-white when night mode was toggled. That value is the same as the page's default.